docs/core/tutorials/testing-library-with-visual-studio-code.md

```diff
@@ -98,20 +88,13 @@ To create the test methods:
 
 1. Save your changes.
 
-1. Run the tests:
+## Build and Run your Tests
 
-   ```dotnetcli
-   dotnet test StringLibraryTest/StringLibraryTest.csproj
-   ```
+1. In **Solution Explorer**, right-click and select **Build** or from the Command Palette, select **.NET: Build**.
 
-   The terminal output shows that all tests passed.
+1. Select the **Testing** window, select **Run Tests** or from the Command Palette, select **Test: Run all Tests**.
 
-   ```output
-   Starting test execution, please wait...
-   A total of 1 test files matched the specified pattern.
-
-   Passed!  - Failed:     0, Passed:     3, Skipped:     0, Total:     3, Duration: 3 ms - StringLibraryTest.dll (net8.0)
-   ```
+   :::image type="content" source="media/testing-library-with-visual-studio-code/testingScreenshot.png" alt-text="Visual Studio Code Test Explorer":::
```

Comment:
The image got put in the wrong place, but we can leave it there and change the reference here.
```suggestion
   :::image type="content" source="testing-library-with-visual-studio-code/testingScreenshot.png" alt-text="Visual Studio Code Test Explorer":::
```